container/openejb-loader/src/main/java/org/apache/openejb/observer/event/ObserverAdded.java

```diff
@@ -26,10 +26,20 @@
 
     private final Object observer;
 
+    /**
+     * Add the provided object to be observable
```

Comment:
That is not what happens here.

Reply:
Same as previously but for adding a new observer in the system. This is also an event